HHS/Head-Start-TTADP
# [TTAHUB-1992] Fix HTML nesting for topics drawer/select

## Description of change

When testing the form with a screenreader, I found the way that the "tell me more" button was nested inside of the <label> element associated with the select to produce confusing output. This change moves the button out of the label but preserve existing styles so that the navigation text isn't confusing.

## How to test

Ensure functionality remains the same. Navigate the controls with a screenreader (on the AR or the RTR) and make sure that the text reads appropriately/makes sense to you.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1922

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
